tensorflow/contrib/nodejs/test/suite.js

```diff
@@ -0,0 +1,7 @@
+var test = require('tape')
+  , tensorflow = require('../index')
```

Comment:
Don't use comma first style, add semi-colons, add space between `(t)` and `{` in the function

I assume TF will want to follow this style guide https://google.github.io/styleguide/javascriptguide.xml ?